.github/workflows/release.yml

```
@@ -0,0 +1,24 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [ created ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: __token__
```

I think you should store it in the GH secrets and retrieve it via

```suggestion
          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
```

---

I wouldn't worry about exposing that you use a `__token__` for the upload. I don't think it really helps people hack anything (if anything, I'd say it discourages it since these are meant to be harder than user passwords to crack)